monai/transforms/spatial/array.py

```diff
@@ -91,7 +91,10 @@ def __init__(
     ) -> None:
         """
         Args:
-            pixdim: output voxel spacing.
+            pixdim: output voxel spacing. if providng a single number, will use it for all dimensions.
+                if the components of the `pixdim` are non-positive values, the transform will use the
+                corresponding components of the origial pixdim, which is computed from the `affine`
+                matrix of input image.
```

Comment:
In old version, if `pixdim` is a sequence, it can has larger length than `sr`, but now we have the restriction: `out_d = ensure_tuple_rep(self.pixdim, sr)`. Therefore, I suggest here to add more docstring about the length of it. Like:
If providing a sequence, please ensure that its length equals to the spatial size of input image.

Reply:
sure, updated in the latest commit.
Thanks.